README.md

```diff
@@ -60,45 +60,40 @@ source $HOME/catkin_ws/devel/setup.bash
 __Usage__
 
 ___With real Hardware___
-There are launch files available to bringup a real robot - either UR5 or UR10.  
-In the following the commands for the UR5 are given. For the UR10, simply replace the prefix accordingly.
 
-Don't forget to source the correct setup shell files and use a new terminal for each command!   
+For using real hardware, please use the
+[`ur_robot_driver`](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver). Installation
```

Comment:
As per #433, this should mention using `ur_modern_driver` for CB1/CB2

Reply:
See f83eb2b :-)